README.md

```diff
@@ -47,6 +47,14 @@ docker:
     - "/path/to/another-image-name.tar"
 ```
 
+### Run a Command
+device-init can execute a list of commands on boot. e.g.:
+```yaml
+runcmd:
+  - "apt-get install package-name"
```

Comment:
According to tests it actually works without quotes in yaml, so we can skip them in here. Go run command should escape this string.